package.json

```diff
@@ -22,16 +22,16 @@
   },
   "homepage": "https://github.com/vtex/eslint-config-vtex#readme",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.12.0",
-    "@typescript-eslint/parser": "^1.12.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-lodash": "^5.1.0",
+    "@typescript-eslint/eslint-plugin": "^2.1.0",
+    "@typescript-eslint/parser": "^2.1.0",
+    "eslint-config-prettier": "^6.2.0",
+    "eslint-plugin-lodash": "^6.0.0",
     "eslint-plugin-prettier": "^3.1.0"
   },
   "devDependencies": {
-    "eslint": "^6.0.1",
+    "eslint": "^6.3.0",
     "prettier": "^1.18.2",
-    "typescript": "^3.5.3"
+    "typescript": "^3.6.2"
```

Comment:
Should we use the same version of TS used in our builders?

Reply:
sure! are we still using 3.5.3?

Reply:
but I think this is only in the dev dependencies, not peer

Reply:
Oh, ok, so I think it might not make any difference